browser-test/src/application_review.test.ts

```diff
@@ -74,18 +79,23 @@ describe('normal application flow', () => {
     await applicantQuestions.answerAddressQuestion('1234 St', 'Unit B', 'Sim', 'Ames', '54321');
     await applicantQuestions.clickNext();
 
-    // Applicant fills out second application block.
+    // fill 2nd application block.
     await applicantQuestions.answerDropdownQuestion('banana');
     await applicantQuestions.answerCheckboxQuestion(['cherry', 'pine']);
     await applicantQuestions.answerNumberQuestion('42');
     await applicantQuestions.answerTextQuestion('some text');
     await applicantQuestions.clickNext();
 
-    // Applicant fills out third application block.
+    // fill 3rd application block.
     await applicantQuestions.answerFileUploadQuestion('file key');
     await applicantQuestions.clickNext();
 
-    // Applicant submits answers from review page.
+    // fill 4th application block. 
+    // skip one checkbox question.
+    await applicantQuestions.answerCheckboxQuestion(['clowns']);
+    await applicantQuestions.clickNext();
+
+    // submit

```

Comment:
Do I need to modify the test to verify that if it skips a non-checkbox question that it results in a review page with 'continue' on it? i.e. to make sure that me adding stuff didn't make it artificially result in 'submit'